File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/model/GenshinGachaModel.java
package com.mcreater.amclcore.api.mihoyo.genshin.model;

import com.mcreater.amclcore.annotations.RequestModel;
import lombok.Data;

@Data
@RequestModel
public class GenshinGachaModel {
    private int retcore;
    private String message;
    private GenshinGachaDataModel data;
}

File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/model/GenshinGachaModel.kt
package com.mcreater.amclcore.api.mihoyo.genshin.model

import com.mcreater.amclcore.annotations.RequestModel

@RequestModel
data class GenshinGachaModel(
    var retcode: Int = 0,
    var message: String? = null,
    var data: GenshinGachaDataModel? = null
)
